README.md

```diff
@@ -1,18 +1,24 @@
 # Dash Sentinel
 
-An all-powerful toolset for Dash.
-
 [![Build Status](https://travis-ci.org/dashpay/sentinel.svg?branch=master)](https://travis-ci.org/dashpay/sentinel)
 
-Sentinel is an autonomous agent for persisting, processing and automating Dash governance objects and tasks, and for expanded functions in the upcoming Dash V13 release (Evolution).
+> An automated governance helper for Dash Masternodes.
+
+Sentinel is an autonomous agent for persisting, processing and automating Dash governance objects and tasks. It is Python application which runs alongside the DashCore instance on each Dash Masternode.
 
-Sentinel is implemented as a Python application that binds to a local version 12 dashd instance on each Dash Masternode.
+## Table of Contents
+- [Install](#install)
+  - [Dependencies](#dependencies)
+- [Usage](#usage)
```

Comment:
1. No Usage section to link to
2. Configuration and Troubleshooting sections are both at a H2 level, but not shown in ToC